pg-extend/src/pg_fdw.rs

```diff
@@ -217,7 +217,7 @@ impl<T: ForeignData> ForeignWrapper<T> {
         row.get_field(&name, typ, opts).map_err(|e| e.into())
     }
 
-    fn tts_to_hashmap(slot: *mut pg_sys::TupleTableSlot, tupledesc: &pg_sys::TupleDesc) -> Tuple {
+    fn tts_to_hashmap(slot: *mut pg_sys::TupleTableSlot, tupledesc: pg_sys::TupleDesc) -> Tuple {
```

Comment:
Curious about this change, generally borrowed variables are correct, does this fix an issue?

Reply:
If you look at the types from Postgres, TupleDesc is just a pointer to tupleDesc. So we were in essence borrowing a raw pointer, which is redundant. Clippy asked me to do away with it.

Reply:
got it.